example/index.js

```diff
@@ -681,9 +681,57 @@ async function updateModel() {
 		material.tangentAttribute.updateFrom( geometry.attributes.tangent );
 		material.uvAttribute.updateFrom( geometry.attributes.uv );
 		material.materialIndexAttribute.updateFrom( geometry.attributes.materialIndex );
+
```

Comment:
@gkjohnson Should this code go into three-mesh-bvh ? https://github.com/gkjohnson/three-mesh-bvh/blob/master/src/gpu/VertexAttributeTexture.js

Reply:
When using the FloatVertexAttributeTexture the values should already be normalized if using an int or uint buffer:

https://github.com/gkjohnson/three-mesh-bvh/blob/master/src/gpu/VertexAttributeTexture.js